src/encoded/schemas/tissue_sample.json

```diff
@@ -100,6 +106,26 @@
             "minimum": 0
         }
     },
+    "allOf": [
+        {
+            "if": {
+                "not": {
+                    "properties": {
+                        "category": {
+                            "const": "Core"
+                        }
+                    }
+                }
+            },
+            "then": {
+                "not": {
+                    "required": [
+                        "core_size"
+                    ]
+                }
+            }
+        }
+    ],
```

Comment:
Would be good to add a quick test for this as well. Also, should it be required to have the core size if it is a core?

Reply:
This has actually introduced a larger issue with how we call `get_identifying_insert` in `tests/utils.py` for testing items. Currently the `get_required_properties` function doesn't handle `allOf` statements or `if/then/else` statements for conditionally required properties. I'm working on a work-around that doesn't use either of these statements to address this for now.

Reply:
I found a workaround using `anyOf` that functions. I've added tests for this and for `target_monomer_size` and `concatenated_reads` in Library.